hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -227,6 +231,9 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
           <Col xs={24} sm={18} md={12} lg={12} xl={6}>
             <OverviewCard loading={loading} title='Keys' data={keys.toString()} icon='file-text'/>
           </Col>
+          <Col xs={24} sm={18} md={12} lg={12} xl={6}>
+            <OverviewCard loading={loading} title='Deleted Containers' data={deletedContainers !== undefined && deletedContainers.toString()} icon='delete' />
```

Comment:
This check seems redundant `deletedContainers !== undefined`, unless there is a chance that the server response won't contain it.

Reply:
Can you do a manual test with docker, without the `pnpm` tool?

Reply:
@xBis7
I checked with docker also working fine with undefined condition.
undefined is additional check if deletedContainers will fail in API Response.

Reply:
I am also not sure why the `deletedContainers !== undefined` check is needed, non of the above `OverviewCard`s contain it.
why can't we assume that the API response will always return this information? the backend part of this have been in done in #4440 and I don't see anything there that would require this check, right @ArafatKhan2198? 
btw I tried in a docker cluster and it was working for me.

Reply:
@dombizita 
Removed undefined condition in latest commit.